components/floatingactionbutton/stories/floatingactionbutton.stories.js

```diff
@@ -4,6 +4,8 @@ import { default as IconStories } from "@spectrum-css/icon/stories/icon.stories.
 
 /**
  * The floating action button component is used to give users a more prominent button for high frequency actions.
+ *
+ * When using floating action button in dark themes, the `background-layer-color-2` will oftens how up on the base color `gray-50` or `gray-75` or on content, images, etc.
```

Comment:
```suggestion
 * When using floating action button in dark themes, the `background-layer-color-2` will often show up on the base color `gray-50` or `gray-75` or on content, images, etc.
```
Other than the typo, I'm not totally sure what the original sentence is trying to say here. So I'm not sure what rewording to suggest.

Reply:
I'm wondering if this is particularly relevant for the secondary button? [At the time that this was added to the documentation](https://pr-1793--spectrum-css.netlify.app/floatingactionbutton#secondary) the secondary button color and background were the same color in dark mode (`background-layer-color-2` or `gray-100`) and the only thing distinguishing the button from the background in dark mode is the drop-shadow. I'm reading it as "in dark mode, the background color `gray-50` or `gray-75` (or content/images) is often used (to make the button show up better)." Thoughts?

Reply:
That does seem to make sense that it's talking about the secondary variant lacking contrast on top the default gray-100 background in dark mode. Maybe see if the design team can clarify or create a followup to improve this later. Does it mean: "When using floating action button in dark themes, the background color `--spectrum-background-layer-2-color` will often show up better on top of a base color of `gray-50` or `gray-75` instead of `gray-100`"? I'm just guessing.